src/content/selectors/index.js

```diff
@@ -187,14 +188,29 @@ export const selectorsForThread = threadIndex => {
         return funcArrays.map(funcArray => ProfileData.getFuncStackFromFuncArray(funcArray, funcStackInfo.funcStackTable));
       }
     );
+    const getFuncStackMaxDepth = createSelector(
+      getFilteredThread,
+      getFuncStackInfo,
+      StackTiming.computeFuncStackMaxDepth
+    );
+    // TODO - Memoize off of the initial profile, ignore changes to the symbol information.
```

Comment:
This is probably not possible - symbolication can make adjacent frames collapse into each other because they turn out to have been in the same function.